modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java

```diff
@@ -59,7 +59,7 @@
     private HashMap<SmallFireball, Long> FireballTracking = new HashMap<>();
     private HashMap<HitBox, Long> wrecks = new HashMap<>();
     private HashMap<HitBox, World> wreckWorlds = new HashMap<>();
-    private HashMap<HitBox, Map<MovecraftLocation,Material>> wreckPhases = new HashMap<>();
+    private HashMap<HitBox, Map<MovecraftLocation, AbstractMap.SimpleImmutableEntry<Material, Byte>>> wreckPhases = new HashMap<>();
```

Comment:
I'd prefer to be consistent with pair objects throughout the project. use `org.apache.commons.lang3.tuple.ImmutablePair` instead